ee/clickhouse/queries/test/test_lifecycle.py

```diff
@@ -184,9 +189,60 @@ def test_interval_dates_months(self):
         )
         self.assertEqual(result[0]["days"], ["2021-02-01", "2021-03-01", "2021-04-01", "2021-05-01"])
 
+    @also_test_with_materialized_columns(event_properties=["int_value"])
+    @snapshot_clickhouse_queries
+    def test_lifecycle_hogql_event_properties(self):
+        with freeze_time("2021-05-05T12:00:00Z"):
+            self._setup_returning_lifecycle_data(20)
+            result = self._run_lifecycle(
+                {
+                    "date_from": "-7d",
+                    "interval": "day",
+                    "properties": [
+                        {"key": "toInt(properties.int_value) > 10 and 'bla' != 'a%sd'", "type": "hogql"},
+                    ],
+                }
+            )
+        assertLifecycleResults(
+            result,
+            [
+                {"status": "dormant", "data": [0] * 8},
+                {"status": "new", "data": [0] * 8},
+                {"status": "resurrecting", "data": [0] * 8},
+                {"status": "returning", "data": [0] * 8},
```

Comment:
Nit: might be more useful to test actually getting some data back?

Reply:
yup, makes sense